cliconfig/config.go

```diff
@@ -32,7 +32,7 @@ var (
 func getDefaultConfigDir(confFile string) string {
 	confDir := filepath.Join(homedir.Get(), confFile)
 	// if the directory doesn't exist, maybe we called docker with sudo
-	if _, err := os.Stat(configDir); err != nil {
+	if _, err := os.Stat(confDir); err != nil {
```

Comment:
entirely my fault, going to make a follow up to clarify this after this get merged